### PR TITLE
Add `RtwithoutRenewalTestPipeline` type with updated pipeline tests

### PR DESCRIPTION
**Pull request by SamuelBrand1**
This small PR follows on from f2f discussion and adds a test mode pipeline type to differentiate between sampling done running tests vs full pipeline runs.

### CODE REVIEW

File: pipeline/src/EpiAwarePipeline.jl [hunk 1]
```diff
@@ -15,7 +15,7 @@ using CSV, Dagger, DataFramesMeta, Dates, Distributions, DocStringExtensions, Dr
 
 # Exported pipeline types
 export AbstractEpiAwarePipeline, EpiAwarePipeline, RtwithoutRenewalPipeline,
-       RtwithoutRenewalPriorPipeline
+       RtwithoutRenewalPriorPipeline, RtwithoutRenewalTestPipeline
```
Comment (seabbs) [inline pipeline/src/EpiAwarePipeline.jl:18]: I thought the plan was to move towards EpiAwarePipeline. Based on that idea shouldn't this be `EpiAwareTestPipeline` or `EpiAwareTodyPipeline`?

**Review by seabbs — APPROVED** [review]
Only comment is about the naming

File: pipeline/src/EpiAwarePipeline.jl (hunk 1 above)
Comment (SamuelBrand1) [reply pipeline/src/EpiAwarePipeline.jl:18]: Yes, good point!

**Comment by seabbs** [comment]
Stochastic failure is negative binomial which is an epiaware issue. Aside from updating the name this is good to go.

**Pull request merged**